modules/social_features/social_post/modules/social_post_album/assets/libraries/slick/fonts/slick.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Couldn't we load this as a proper library instead? Now we need to update all individual files once slick releases something.